lib/charms/opensearch/v0/opensearch_config.py

```diff
@@ -125,6 +125,12 @@ def set_node_tls_conf(self, cert_type: CertType, truststore_pwd: str, keystore_p
                 f"plugins.security.ssl.{target_conf_layer}.{store_type}_password",
                 pwd,
             )
+            if store_type == "keystore":
```

Comment:
better move this one out of the `for` loop and directly target the corresponding key